apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -100,8 +100,52 @@ export async function processPageResearch({ blocks }) {
   });
 }
 
+export async function processPageAbout({ blocks }) {
+  blocks.push({
+    slug: "grantees",
+    title: "Grantees",
+    grantees: Array.from({ length: 30 }, (_, i) => ({
+      id: i,
+      name: "Grantee Name ".repeat((i % 2) + 1).trim(),
+      description: [
+        {
+          children: [
+            {
+              text: "Lorem ipsum dolor sit amet con sectetur adipiscing elit mi, interdum blandit fring illa fus. adipiscing elit mi, adipiscing.",
+            },
+          ],
+        },
+      ],
+      image: {
+        id: "63d2622aafe25f6469605eae",
+        alt: `Research ${i}`,
+        prefix: "media",
+        filename: "Rectangle 113.jpg",
+        mimeType: "image/jpg",
+        filesize: 257010,
+        width: 1236,
+        height: 696,
+        createdAt: "2023-01-26T11:21:14.868Z",
+        updatedAt: "2023-01-26T11:21:14.868Z",
+        url: "http://localhost:3000/media/Rectangle 117.png",
```

Comment:
I think it should refer to this [image](https://github.com/CodeForAfrica/ui/blob/7e9d370eef9c39f650bd649d5a59403c6eef3fa3/apps/charterafrica/public/images/the-charter-project-africa_brandguide_guidedemarque-2-2.jpg)
Also use relative path i.e `/images/....` incase someone runs it using a different port